base/applications/notepad/lang/ro-RO.rc

```diff
@@ -136,12 +140,12 @@ BEGIN
 END
 
 DIALOG_PRINTING DIALOG 0, 0, 160, 100
-CAPTION "Now printing"
+CAPTION "În curs de imprimare"
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CTEXT "Print job is starting...", IDC_PRINTING_STATUS, 5, 10, 150, 15
-    CTEXT "(Filename)", IDC_PRINTING_FILENAME, 5, 35, 150, 15
+    CTEXT "Începe sarcina imprimării...", IDC_PRINTING_STATUS, 5, 10, 150, 15
+    CTEXT "(Numele fișierului)", IDC_PRINTING_FILENAME, 5, 15, 150, 15
```

Comment:
Is this Y-coordinate change intentional? (35 => 15)

Reply:
Sorry, I will revert the Y-coordinate. I mistakenly changed it.